code/modules/vending/games.dm

```diff
@@ -11,7 +11,8 @@
 		            /obj/item/toy/cards/deck/cas/black = 3,
 		            /obj/item/toy/cards/deck/unum = 3,
 					/obj/item/toy/cards/deck/tarot = 3,
-					/obj/item/hourglass = 2)
```

Comment:
This minor change is unrelated to the PR and should be reverted.

Also how does this hanging comma not cause issues 🤔

Reply:
Hanging commas are standard in lists like this so you don't have to make unnecessary changes to add a new line.